biz.aQute.bndlib/src/aQute/bnd/differ/JavaElement.java

```diff
@@ -299,9 +299,7 @@ public void implementsInterfaces(TypeRef names[]) throws Exception {
 				Arrays.sort(names); // ignore type reordering
 				for (TypeRef name : names) {
 
-					String comment = null;
-					if (clazz.isInterface() || clazz.isAbstract())
-						comment = inherit(members, name);
+					String comment = inherit(members, name);
 					members.add(new Element(IMPLEMENTS, name.getFQN(), null, MINOR, MAJOR, comment));
```

Comment:
Drop the temporary reference and just pass the result of `inherit()` directly to the `Element` constructor